app/lib/pdf_filler/ny201_pdf.rb

```diff
@@ -125,8 +125,8 @@ def hash_for_pdf
       },
       'Filing_status' => {
         1 => '1 Single',
-        2 => '2 Married Filing Joint Return (enter spouse’s social security number above)',
-        3 => '3 Married Filing Seperate Return (enter spouse’s social security number above)',
+        2 => '2 Married Filing Joint Return',
+        3 => '3 Married Filing Separate Return',
```

Comment:
the apostrophes in these options was causing a [bug](https://www.pivotaltracker.com/story/show/186100685) where we couldn't check the box. i fixed it by updating the values for these options in the new pdf, with permission from NY.

Reply:
thank you for pdf form wrangling!